### PR TITLE
Test for editing tasks.

### PR DESCRIPTION
**Pull request by addisalemseifu**
-Added test for the `clearDone()` method.
-Added test for the `checkDone()` method.
-Added test for the `makeEdditable()` method.
-Ignored the `--coverage`.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -35,12 +35,38 @@ document.querySelector('.addBook').addEventListener('click', (e) => {
...
   // Eddit task.
   if (e.target.classList.contains('tasks')) {
-    Store.update(e.target);
+    const ide = e.target.parentElement.firstChild.id;
+    e.target.removeAttribute('readonly');
+    e.target.addEventListener('input', () => {
+      const tasks = Store.getTask();
+      // eslint-disable-next-line arrow-body-style
+      const matching = tasks.find((task) => {
+        // eslint-disable-next-line eqeqeq
```
Comment (OLIPLICHE) [inline src/index.js]: - [ ] Going through this file, I found out that you disabled the Linters all over the codes. I would advise you not to disable them, you can solve them manually AND satisfy the general requirements.

File: src/storage.js [hunk 2]
```diff
@@ -32,34 +32,15 @@ export default class Store {
...
   }
 
-  static markDone = (id, target) => {
+  static markDone = (id, insStatus) => {
     const tasks = Store.getTask();
-    // eslint-disable-next-line arrow-body-style
-    const matching = tasks.find((task) => {
-      // eslint-disable-next-line eqeqeq
-      return task.index == id;
-    });
```
Comment (OLIPLICHE) [inline src/storage.js:56]: - [ ] Going through this file, I found out that you disabled the Linters all over the codes. I would advise you not to disable them, you can solve them manually AND satisfy the general requirements.

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Created test files ([..].test.js) which contain the add item and delete item 👍 
- Tests are grouped using the `description()` method. 👍 
- README file is descriptive 👏

## Required Changes ♻️

- [ ] Hi, The Project requirement says you have to use mock for functions that used `localStorage`, Can you provide this to satisfy the requirement?
- [ ] In ReadMe Template, you provided two live demo links and one of them is not working. I would advise you to remove the one which is not working to avoid confusion, thus making your template more professional.
- [ ] Going through your files, I found out that you disabled the Linters from `ui.js`, `storage.js`, and `index.js`. I would advise you not to disable them, you can solve them manually AND to satisfy the general requirements:
- A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. [Here are many reasons not to disable the linter](https://github.com/microverseinc/linters-config). 

- [ ] NB: Some of the comments above are not in line with the code because you did not make the files/code visible in this PR.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by addisalemseifu** [comment]
-Hey there thanks for all the comments. I have fixed all of them but on the part where you asked me to `mock` the `localStorage` I am using the latest version of `jest` and the requirement requires me to mock the local storage only if I am using the older version of `jest`.
If you have any suggestions, feel free to forward them. 
Thanks in advance

**Comment by amon-cofie** [comment]
> -Hey there thanks for all the comments. I have fixed all of them but on the part where you asked me to `mock` the `localStorage` I am using the latest version of `jest` and the requirement requires me to mock the local storage only if I am using the older version of `jest`. If you have any suggestions, feel free to forward them. Thanks in advance

👍 If you have the latest version of jest you don't have to mock the local storage since it has already been mocked by jest in their latest version. You just have to use the local storage like you would in any of your functions to persist data.

**Review by amon-cofie — CHANGES_REQUESTED** [review]
Hi @addisalemseifu and @Amazinggracee  , 👋 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.




 ## Highlights

🚀 You have a descriptive pull request summary
🚀 You have good commit messages
🚀 You used GitHub flow properly
🚀 You have no linter errors


 ## Required Changes ♻️

You have done a very good job on the testing 👍 . However, it seems 2 of your tests are failing. Please check the screenshots below to find out which ones: 
image


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @addisalemseifu and @amon-cofie 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Use jest framework for testing ✔️ 
All test passed ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**